src/resources/subscription_entitlement.ts

```diff
@@ -9,6 +9,7 @@ export class SubscriptionEntitlement extends Model {
   public subscription_id: string;
   public feature_id?: string;
   public feature_name?: string;
+  public feature_type?: 'switch' | 'quantity' | 'range' | 'custom';
```

Comment:
from what I've experienced, this string is capitalized. 
```suggestion
  public feature_type?: 'SWITCH' | 'QUANTITY' | 'RANGE' | 'CUSTOM';
```

Reply:
Maybe Chargebee changed it again...

Reply:
Right, if this is a difference between PC v1 and PC v2, it would be great if someone from Chargebee could clarify and resolve this.